index.js

```diff
@@ -2,7 +2,24 @@ const readPkgUp = require('read-pkg-up');
 const LastCommitLog = require('last-commit-log');
 
 /**
- * Information about the app
+ * A safe way to query for git infos.
+ *
+ * @return {object|undefined} Returns {hash, gitTag} or undefined in case of an error
+ */
+async function getLastCommit() {
+  try {
+    const lastCommitLog = new LastCommitLog();
+    const { hash, gitTag } = await lastCommitLog.getLastCommit();
+    return { hash, gitTag };
+  } catch (err) {}
```

Comment:
Can you change it so we can pass a custom logger?

```diff
-async function parseAppInfo() {
+async function parseAppInfo(logger = console) {
```

Then make it so `getLastCommit` accepts this logger as well?

```diff
-async function getLastCommit() {
+async function getLastCommit(logger) {
  try {
    const lastCommitLog = new LastCommitLog();
    const { hash, gitTag } = await lastCommitLog.getLastCommit();
    return { hash, gitTag };
-  } catch (err) {}
+  } catch (err) {
+    logger.error(err);
+    return {};
  }
}
```